Cargo.toml

```diff
@@ -55,18 +55,20 @@ lightning-liquidity = { version = "=0.1.0-alpha.4", features = ["std"] }
 #lightning-transaction-sync = { path = "../rust-lightning/lightning-transaction-sync", features = ["esplora-async"] }
 #lightning-liquidity = { path = "../lightning-liquidity", features = ["std"] }
 
-bdk = { version = "0.29.0", default-features = false, features = ["std", "async-interface", "use-esplora-async", "sqlite-bundled", "keys-bip39"]}
+bdk_chain = { version = "0.18.0", default-features = false, features = ["std"] }
+bdk_esplora = { version = "0.17.0", default-features = false, features = ["async-https-rustls"]}
+bdk_wallet = { version = "1.0.0-beta.2", default-features = false, features = ["std", "keys-bip39"]}
```

Comment:
It may be worthwhile to upgrade to `1.0.0-beta.4` and the respective `bdk_chain` and `bdk_esplora` updates that were released today. I do not think there are any breaking changes, at least in the `bdk_wallet` API